### PR TITLE
#150798553 Notify a user when a job runs too often

### PR DESCRIPTION
**Pull request by rosemaina**
What does this PR do?
#### Description of Task to be completed?
-As a user I want to receive an email when a job is run too often
-As a user I want an icon on the dashboard indicating that a job is run too often
#### How should this be manually tested?
-After cloning the repo cd into it and run this command `./manage.py runserver`
-In your terminal run this command `./manage.py sendalerts`
NB:Run the two commands at the same time, ie in different tabs
#### What are the relevant pivotal tracker stories?
#150798553
#### Screenshots (if appropriate)
screen shot 2017-09-14 at 14 28 23

### CODE REVIEW

**Comment by judithachieng** [comment]
@rosemaina @borenho 59 commits?  Please . squash them

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```
Comment (judithachieng) [inline .idea/vcs.xml]: .idea/ should not be pushed to git . @rosemaina @borenho

**Comment by rosemaina** [comment]
@andela-jmwalo 
I have checked the commits...some are like 10 days old
But i have squashed the commits I made

File: .gitignore [hunk 2]
```diff
@@ -13,5 +13,5 @@ static/.DS_Store
 .idea/healthchecks-invincibles.iml
 .idea/misc.xml
 .idea/modules.xml
-.idea/workspace.xml
+.idea/worksgpace.xml
```
Comment (judithachieng) [inline .gitignore]: this should be .idea/workspace why change it to .idea/workgspace?

Comment (judithachieng) [reply .gitignore]: @rosemaina @borenho

File: requirements.txt [hunk 3]
```diff
@@ -64,4 +79,4 @@ virtualenvwrapper==4.7.2
 wcwidth==0.1.7
 webencodings==0.5.1
 zmq==0.0.0
-sendgrid-django
+
```
Comment (judithachieng) [inline requirements.txt]: Why have you removed  `sendgrid-django`,  `psycopg2==2.6.1` and `coveralls==1.2.0` from requirements ?? @borenho @rosemaina

**Pull request merged**